standard/lexical-structure.md

```diff
@@ -459,6 +505,8 @@ Contextual_Keyword
     ;
 ```
 
+> *Note*: The rules `keyword` and `contextual_keyword` are parser rules as they do not introduce new token kinds. Every keyword and contextual keyword is defined by an implicit lexical rule as they occur as literal strings in the grammar ([§7.2.3](lexical-structure.md#723-lexical-grammar)). *end note*
```

Comment:
Suggested pluralization consistency:

"All keywords and contextual keywords are defined by implicit lexical rules as they occur as literal strings in the grammar"

Reply:
Done